variables.tf

```
@@ -39,6 +39,16 @@ variable "dependency_ids" {
   default = {}
 }
 
+variable "app_autosync" {
+  type    = bool
+  default = true
+}
+
+variable "target_revision" {
+  description = "Override of target revision of the application chart."
+  type        = string
+  default     = "main" # x-release-please-version
```

This cannot be main here. You should point to the tag version!
```suggestion
  default     = "1.0.0-alpha.2" # x-release-please-version
```

Release-please will take care of updating this file since there is a comment `# x-release-please-version` on the line.